colorcutquantizer.go

```diff
@@ -20,16 +20,15 @@ const (
 type colorCutQuantizer struct {
 	Colors           []int
 	ColorPopulations map[int]int
+	ColorRatios      map[int]float64
 	QuantizedColors  []*Swatch
+	PixelCount       int
 }
 
-// true if the color is close to pure black, pure white, or
-// "the red side of the I line" which I believe is Google-speak for
-// "that particular shade of red which occurs in the red-eye effect"
-// see enwp.org/Red-eye_effect
+// true if the color is close to pure black, pure white
 func shouldIgnoreColor(color int) bool {
-	h, s, l := rgbToHsl(color)
-	return l <= blackMaxLightness || l >= whiteMinLightness || (h >= 0.0278 && h <= 0.1028 && s <= 0.82)
```

Comment:
we don't care about red eye, this seem a feature targeting consumer photography and may bit us back excluding some red shades